app/config/question-bank.js

```diff
@@ -3948,7 +3934,7 @@ const questionBank = {
             {
               type: 'REGEX',
               // only 2 decimal places
-              regex:  /^(\d{1,2}(\.\d{1,2})?)$/,
+              regex:  /^(\d{1,9}(\.\d{1,2})?)$/,
```

Comment:
We have the following regex already in common functions, would this work instead?

const TWO_DP_NUMBER = /^\d+(\.?)(\d{0,2})$/